backend/src/interfaces/Ientities/Icourses.ts

```diff
@@ -5,7 +5,7 @@ import { UserRank } from 'enums/users';
 interface ICourse {
   _id?: ObjectId;
   title: string;
-  technologies: ObjectId[];
+  technologies: { skill: ObjectId; points: number }[];
   requiredSkills?: ObjectId[];

```

Comment:
We don't use `requiredSkills` anywhere. If @k-krasilnikova agree, I would propose to remove this field from models, interfaces and migrations.

Reply:
Yep, let's do it in a separate PR @SaXaPhonist @dapodev